src/actions/events.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+const dbUrl = process.DATABASE_URL || "http://localhost:4000";
```

Comment:
If you are using axios you can create an instance of it in a separate file with configurations and then export that instance and use that in your action files, like:
```
// src/axios.js
import axios from "axios";

const instance = axios.create({
  baseURL: "http://localhost:4000"
});
export default instance;
```

Reply:
```
import axios from '../axios.js`

axios.post('/event', { body })
```